src/test/Figure/Population.py

```diff
@@ -0,0 +1,29 @@
+from typing import Optional
+
+from converter import CamelCaseModel, DataProductDefinition
+from pydantic import Field
+
+
+class PopulationResponse(CamelCaseModel):
+    label: str = Field(..., title="Data reference label")
+    source: str = Field(..., title="Data source name")
+    value: int = Field(..., title="The population value")
+    updated: str = Field(..., title="Data updated at timestamp text")
+
+
+class PopulationRequest(CamelCaseModel):
+    # country: Optional[str] = Field(None, title="Country")
+    city: Optional[str] = Field(
+        "",
+        title="City or region",
+        description="City or a region name, leaving the field empty selects country's total",
+    )
+    year: Optional[int] = Field(2021, title="Year")
```

Comment:
I believe you would actually want to define this as an `int`, not an `Optional[int]`. If you define it as an optional int, a user is able to send a payload where it's actually set to null/None (not the default value), which I don't think is desireable (I might of course be wrong about this). If you use `int`, then the `year` parameter can be left out from the request and default to 2021, but it can't be included and set to `null`.